editor.html

```diff
@@ -395,7 +395,7 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
             <div id="editor" class="flex3" tabindex="-1"></div>
           </div>
         <div id="flashing-overlay-container" class="modal-overlay-container">
-          <div id="flashing-overlay" class="modal-overlay" tabindex="-1" role="dialog" aria-labelledby="flashing-overlay" aria-modal="true">
+          <div id="flashing-overlay" class="modal-overlay modal-div" tabindex="-1" role="dialog" aria-labelledby="flashing-overlay" aria-modal="true">
```

Comment:
Is adding `modal-div` these necessary since focusModal now takes the element ID?
If not, I'd rather not include it as it might introduce subtle CSS changes.
Same goes for `modal-msg-overlay`.

Reply:
I think it might be neccessary, as without it I get a focus ring around each modal, which is what we wanted to avoid in https://github.com/bbcmicrobit/PythonEditor/pull/290#issuecomment-548026796
<img width="439" alt="Screenshot 2019-12-10 at 14 34 49" src="https://user-images.githubusercontent.com/31242877/70538615-511bbb80-1b5a-11ea-80af-3bb08ac69074.png">
<img width="1084" alt="Screenshot 2019-12-10 at 14 34 32" src="https://user-images.githubusercontent.com/31242877/70538619-52e57f00-1b5a-11ea-80d2-03254f1fce7b.png">

Reply:
Okay, taking in consideration this is the only thing it is doing, then its fine

```
.modal-div:focus {
    outline: 0;
}
```